storage/remote/azuread/azuread_config.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azuread
+
+import (
+	"fmt"
+
+	"github.com/google/uuid"
+)
+
+// AzureADConfig is the configuration for getting the accessToken
+// for remote write requests to Azure Monitoring Workspace
+type AzureADConfig struct {
+	// ClientId is the clientId of the managed identity that is being used to authenticate.
+	ClientId string `yaml:"client_id,omitempty"`
+
+	// Cloud is the Azure cloud in which the service is running. Example: AzurePublic/AzureGovernment/AzureChina
+	Cloud string `yaml:"cloud,omitempty"`
+}
+
+// Used to validate config values provided
```

Comment:
It would be nice if these comments (and in other files) followed the docstring format, e.g. `Validate validates the config values provided.` https://tip.golang.org/doc/comment. Specifically that they should start with the function name and be complete sentences with a full stop.

Reply:
Make sense. Updated all the comments to follow the style you suggested.